### ISSUE TITLE
Code block syntax highlighting incorrectly

### ISSUE THREAD

**Issue by EllieMakin** [atom-community/markdown-preview-plus#511]
Sometimes (in confusing circumstances) the syntax highlighting for a code block inside the preview gets the colours wrong, but plain markdown-preview renders them correctly.

The problem occurs regardless of whether I am using the pandoc or markdown-it backend.

I have noticed that this happens in several different grammars, but in this particular example, I am using the [language-riscv](https://atom.io/packages/language-riscv) package. Here is the source markdown:

~~~
```riscv
add   s0, a0, t0      #comment (not right)
lw    s2 8(s0)        #comment (renders correctly)
bgeu  s3, s2, 3       #comment (rest of line not right either)
```
~~~

The result in markdown-preview:
![mp](https://user-images.githubusercontent.com/20400201/97240828-3c362e00-17e7-11eb-8fc4-c6c84203d6f5.png)

And the result in markdown-preview-plus:
![mpp](https://user-images.githubusercontent.com/20400201/97240866-553edf00-17e7-11eb-8d81-c8ad97f9bbd4.png)

**Comment by lierdakil** [comment]
Okay, a quick fix: use legacy highlighter
![image](https://user-images.githubusercontent.com/7275622/98020704-6bb2ef00-1e14-11eb-9fbf-9c59d60f973c.png)
meanwhile I'll try to figure out what breaks.

**Comment by lierdakil** [comment]
Should be fixed in v4.8.3. Thanks for reporting!
